Documentation/CRDs/Shared-Filesystem/ceph-filesystem-crd.md

```diff
@@ -135,13 +135,13 @@ spec:
     # for more details see: https://docs.ceph.com/en/latest/dev/cephfs-mirroring/#bootstrap-peers
     peers:
       secretNames:
-        - secondary-cluster-peer
+    # Add the secret name if it already exists else specify the empty list here.
```

Comment:
The comment should be in line with spaces, `do a tab`

And why remove the secret name, let's keep it as for the first time it wouldn't be present.

Reply:
In the docs we have mentioned `Once mirroring is enabled, Rook will by default create its own [bootstrap peer token](https://docs.ceph.com/en/latest/dev/cephfs-mirroring/?#bootstrap-peers) so that it can be used by another cluster.`

Reply:
if we mention the secret name the default secret `fs-peer-token-myfs` is not created.